crowbar_framework/app/models/provisioner_service.rb

```diff
@@ -143,7 +143,7 @@ def transition(inst, name, state)
         else
           node_barclamp = node_role
         end
-        bc_databag = ProposalObject.find_data_bag_item("barclamps/#{node_barclamp}")
+        bc_databag = Chef::DataBag.load("barclamps/#{node_barclamp}") rescue nil
```

Comment:
That's not correct (and broken with the move away from data bags).

When we discussed it the other day, though, we found out that we can just drop this whole block (`if state == "installing"`) as no barclamp is using a `unsupported_platform` attribute.